lib/sqlalchemy/dialects/oracle/base.py

```diff
@@ -46,28 +45,27 @@
 
 The :class:`_schema.Identity` object support many options to control the
 "autoincrementing" behavior of the column, like the starting value, the
-incrementing value, etc.
-In addition to the standard options, Oracle supports setting
-:paramref:`_schema.Identity.always` to ``None`` to use the default
-generated mode, rendering GENERATED AS IDENTITY in the DDL.
-Oracle also supports two custom options specified using dialect kwargs:
+incrementing value, etc.  In addition to the standard options, Oracle Database
+supports setting :paramref:`_schema.Identity.always` to ``None`` to use the
+default generated mode, rendering GENERATED AS IDENTITY in the DDL.  Oracle
+Database also supports two custom options specified using dialect kwargs:
 
 * ``oracle_on_null``: when set to ``True`` renders ``ON NULL`` in conjunction
   with a 'BY DEFAULT' identity column.
 * ``oracle_order``: when ``True``, renders the ORDER keyword, indicating the
   identity is definitively ordered. May be necessary to provide deterministic
   ordering using Oracle RAC.
```

Comment:
below the acronym was expanded. if we want to expand it only one it may make sense to do that here since it's the first one.

(just a FYI, it's fine wither way)

Reply:
I can fix it.
BTW, there are unrelated build issues:

Reply:
fist one is a mystery. never figured out what it was. thanks for mentioning the other two, I'll fix those